x-pack/test/api_integration/apis/security/privileges.ts

```diff
@@ -55,7 +55,7 @@ export default function ({ getService }: FtrProviderContext) {
         'file_operations_all',
         'execute_operations_all',
       ],
-      uptime: ['all', 'read', 'minimal_all', 'minimal_read'],
+      uptime: ['all', 'enable_public_locations', 'read', 'minimal_all', 'minimal_read'],
```

Comment:
Nit: Maybe use a similar naming convention to other existing privileges (item name followed by action/access), which would be something like `public_locations_read`.